connection.go

```diff
@@ -189,7 +189,7 @@ func (c *Connection) sendQuery(q Query) error {
 	// Build query
 	b, err := json.Marshal(q.Build())
 	if err != nil {
-		return RQLDriverError{rqlError("Error building query")}
+		return RQLDriverError{rqlError(err.Error())}
```

Comment:
Perhaps `return RQLDriverError{rqlError(fmt.Sprintf("Error building query: %s", err))}` would make more sense, what do you think?